docs/en/index.md

```diff
@@ -124,3 +124,36 @@ You can disable the native driver by this option (and the emulated will take con
 redis:
 	session: {native: off}
 ```
+
+## Key prefixes
+
+When you use two instances of one application with one Redis server, it is possible, that your data can be overwritten by second application instance.
+
+To avoid this problem you can define key prefixes in configuration, for example:
+
+Instance 1
+```yml
+redis:
+    keyPrefix: "instance1_"
```

Comment:
namespace sounds better to me, [doctrine cache also uses it](https://github.com/doctrine/cache/blob/e0ef9e9833ca024d251e77e8d82a00ca92b0aa71/lib/Doctrine/Common/Cache/CacheProvider.php#L57-L61)